cpp/version.cc

```diff
@@ -0,0 +1,12 @@
+#include "version.h"
+
+#ifndef BUILD_VERSION
+#error Must specify -DBUILD_VERSION=... when building this file.
+#endif
+
+namespace cert_trans {
+
```

Comment:
Extra blank line here or remove one of the two on lines 8 & 9 in the `version.h` file